api/src/game_state_server.py

```diff
@@ -30,9 +31,9 @@ def assign_colors(tokens: List[Token]) -> None:
     for token in tokens:
         if not token.color_rgb:
             if not available_colors:
-                print(f'Max colors reached for icon ' f'{content_id(token.contents)}')
+                logger.info(f'Max colors reached for icon {content_id(token.contents)}')
```

Comment:
logging.getLogger(__name__) should be above this.

Reply:
good point, done